src/aggregation/bucket/histogram/histogram.rs

```diff
@@ -215,6 +216,33 @@ impl SegmentHistogramCollector {
         self,
         agg_with_accessor: &BucketAggregationWithAccessor,
     ) -> crate::Result<IntermediateBucketResult> {
+        // Compute the number of buckets to validate against max num buckets
+        // Note: We use min_doc_count here, but it's only an estimate here, since were are on the
```

Comment:
```suggestion
        // Note: We use min_doc_count here, but it's only an upperbound here, since were are on the
```

It's an upperbound, correct?

Reply:
yes